roles/satellite-clone/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+# more info about these variables can be found in satellite-clone-vars.sample.yml in the root directory of this project
+hostname: satellite.example.com
+satellite_version: 6.1
+perform_restore: true
+backup_dir: /backup
+include_pulp_data: true
+activationkey: changeme
+org: 111111
+register_to_portal: true
+required_root_free_space: 75
+disable_firewall: false
+run_katello_reindex: false
+run_pre_install_check: true
```

Comment:
For easy reference, the order of these variables should be the same as in satellite-clone-vars.sample.yml.